.github/workflows/build.yml

```diff
@@ -0,0 +1,24 @@
+name: Build the candy-board project
+
+on:
+  push:
+    branches:
+    - main
+  release:
+    types: [created, published]
+
+jobs:
+  build_project:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v3
```

Comment:
Can you specify a name for each major step of your workflow ? To ensure to have correctly formatted logs